tests/jballerina-integration-test/src/test/resources/configurables/negative_tests/configProject/main/main.bal

```diff
@@ -26,8 +26,8 @@ type UserTable table<AuthInfo> key(username);
 configurable int intVar = 5;
 configurable string stringVar = ?;
 configurable int[] & readonly intArr = [11, 33];
-configurable AuthInfo & readonly testUser = ?;
-configurable UserTable & readonly users = ?;
+configurable AuthInfo & readonly testUser = {username: "Anna"};
```

Comment:
We moved the required value check to `ConfigTomlParser`. So having many required configurables needs value to be given in all depending toml files. Therfore, changed them to defaultable ones.